CMSgov/mint-app
# [EASI-4170] FE translation export

# EASI-4170


## Changes and Description

- Added ts-node to compile ts translations file for export to JSON
- Added export file to generate json files to `mappings/translation` directory for each section
- Added util for transforming data before export
  - Removing unneeded fields, remapping gql fields to database fields
- Added unit tests



## How to test this change

Run `yarn ts-node ./mappings/export/exportTranslation.ts` to generate JSON files
Verify JSON formatted properly, as well as dbField names as key
Unit tests pass

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
